Jenkinsfile_CNP

```diff
@@ -53,6 +53,13 @@ def vaultOverrides = [
     'preview' : 'aat',
     'spreview': 'saat'
 ]
+// Vars needed for AKS testing
```

Comment:
can make it a bit more explicit please:
Vars needed for functional and smoke tests run against AKS